ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/KnownPeers.hs

```diff
@@ -354,15 +367,16 @@ aboveTarget PeerSelectionPolicy {
     -- below the target for root peers.
     --
   , let numRootPeersCanForget = LocalRootPeers.size localRootPeers
-                              + Set.size publicRootPeers
+                              + PublicRootPeers.size publicRootPeers
```

Comment:
I understood from a chat we had a couple of weeks ago that the current public root peers won't be removed, so we need to keep `targetNumberOfRootPeers`, since these targets are related to the public root peers that are configured in the topology file. Right now `BootstrapPeers` are exactly the same as the `PublicRootPeers` in the topology file. Since we haven't yet decided on the design of the bootstrap peers in the topology file.

Reply:
If bootstrap peers are disabled we should use the old public root peers configuration, right?

Reply:
Also I think we still need a target for the node to fetch new/more ledger/bootstrap peers if it fell short of it (e.g. during churn)

Reply:
I just got confused by the language. We need `targetNumberOfRootPeers` for `ledgerPeers`.

Bootstrap peers need to be something else in the topology file than public root peers.  We should still use public root peers if bootstrap peers are disabled in the same way as we used to use them, e.g. when we are before a configured slot we get ledger peers from the `PublicRootPeers`. Isn't that what we agreed on when we were discussing [this](https://github.com/input-output-hk/ouroboros-network/discussions/4658) last time.

Reply:
> If bootstrap peers are disabled we should use the old public root peers configuration, right?
yes